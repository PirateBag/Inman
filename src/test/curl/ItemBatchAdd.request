{ "updatedRows" : [
  { "summaryId" : "W-001", "description" : "36 In Red Wagon", "unitCost" : "0.0", "sourcing" : "MAN", "activityState" : "INSERT" },
  { "summaryId" : "W-002", "description" : "Red Wagon Body", "unitCost" : "0.0", "sourcing" : "MAN", "activityState" : "INSERT" },
  { "summaryId" : "W-003", "description" : "Front Wheel Assembly", "unitCost" : "0.0", "sourcing" : "MAN", "activityState" : "INSERT" },
  { "summaryId" : "W-004", "description" : "Rear Wheel Assembly", "unitCost" : "0.0", "sourcing" : "MAN", "activityState" : "INSERT" },
  { "summaryId" : "W-006", "description" : "Red Wagon Body", "unitCost" : "0.0", "sourcing" : "MAN", "activityState" : "INSERT" },
  { "summaryId" : "W-007", "description" : "Washer", "unitCost" : ".21", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-008", "description" : "Threaded bolt", "unitCost" : "0.6", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-009", "description" : "Nut", "unitCost" : "0.17", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-010", "description" : "8 In Wheel", "unitCost" : "0.17", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-011", "description" : "Front Wheel Bracket", "unitCost" : "0.95", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-012", "description" : "Yoke", "unitCost" : "0.17", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-005", "description" : "Axle", "unitCost" : "0.0", "sourcing" : "MAN", "activityState" : "INSERT" },
  { "summaryId" : "W-017", "description" : "Rod Stock", "unitCost" : "0.70", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-013", "description" : "Wagon Body", "unitCost" : "2.25", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-014", "description" : "Red Paint", "unitCost" : "20.0", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-015", "description" : "Rear Wheel bracket", "unitCost" : "1.0", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-016", "description" : "Axle Bracket Brace", "unitCost" : "0.75", "sourcing" : "PUR", "activityState" : "INSERT" },
  { "summaryId" : "W-018", "description" : "Carrige Bolt", "unitCost" : "0.17", "sourcing" : "PUR", "activityState" : "INSERT" }
] }
