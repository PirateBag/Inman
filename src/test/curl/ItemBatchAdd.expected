{
  "responseType" : "MULTILINE",
  "errors" : [ ],
  "data" : [ {
    "id" : 1,
    "activityState" : "NONE",
    "summaryId" : "W-001",
    "description" : "36 In Red Wagon",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 2
  }, {
    "id" : 2,
    "activityState" : "NONE",
    "summaryId" : "W-002",
    "description" : "Red Wagon Body",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 10
  }, {
    "id" : 3,
    "activityState" : "NONE",
    "summaryId" : "W-003",
    "description" : "Front Wheel Asmbly",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 1
  }, {
    "id" : 4,
    "activityState" : "NONE",
    "summaryId" : "W-004",
    "description" : "Rear Wheel Asmbly",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 1
  }, {
    "id" : 5,
    "activityState" : "NONE",
    "summaryId" : "W-005",
    "description" : "Axle",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 4
  }, {
    "id" : 6,
    "activityState" : "NONE",
    "summaryId" : "W-006",
    "description" : "Red Wagon Body",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 1
  }, {
    "id" : 7,
    "activityState" : "NONE",
    "summaryId" : "W-007",
    "description" : "Washer",
    "unitCost" : 0.21,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 5
  }, {
    "id" : 8,
    "activityState" : "NONE",
    "summaryId" : "W-008",
    "description" : "Threaded bolt",
    "unitCost" : 0.6,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 5
  }, {
    "id" : 9,
    "activityState" : "NONE",
    "summaryId" : "W-009",
    "description" : "Nut",
    "unitCost" : 0.17,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 6
  }, {
    "id" : 10,
    "activityState" : "NONE",
    "summaryId" : "W-010",
    "description" : "8 In Wheel",
    "unitCost" : 0.17,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 10
  }, {
    "id" : 11,
    "activityState" : "NONE",
    "summaryId" : "W-011",
    "description" : "Front Wheel Bracket",
    "unitCost" : 0.95,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 2
  }, {
    "id" : 12,
    "activityState" : "NONE",
    "summaryId" : "W-012",
    "description" : "Yoke",
    "unitCost" : 0.17,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 10
  }, {
    "id" : 13,
    "activityState" : "NONE",
    "summaryId" : "W-013",
    "description" : "Wagon Body",
    "unitCost" : 2.25,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 20
  }, {
    "id" : 14,
    "activityState" : "NONE",
    "summaryId" : "W-014",
    "description" : "Red Paint",
    "unitCost" : 20.0,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 5
  }, {
    "id" : 15,
    "activityState" : "NONE",
    "summaryId" : "W-015",
    "description" : "Rear Wheel bracket",
    "unitCost" : 1.0,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 4
  }, {
    "id" : 16,
    "activityState" : "NONE",
    "summaryId" : "W-016",
    "description" : "Axle Bracket Brace",
    "unitCost" : 0.75,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 6
  }, {
    "id" : 17,
    "activityState" : "NONE",
    "summaryId" : "W-017",
    "description" : "Rod Stock",
    "unitCost" : 0.7,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 5
  }, {
    "id" : 18,
    "activityState" : "NONE",
    "summaryId" : "W-018",
    "description" : "Carrige Bolt",
    "unitCost" : 0.17,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 3
  } ]
}